apps/qubit/modules/repository/templates/_contextMenu.php

```diff
@@ -1,9 +1,9 @@
 <?php echo get_component('repository', 'logo') ?>
 
-<?php if (QubitAcl::check($resource, 'update')): ?>
-  <?php include_component('repository', 'uploadLimit', array('resource' => $resource)) ?>
-<?php endif; ?>
-
 <?php if ($resource->getRawValue() instanceof QubitRepository): ?>
+  <?php if (QubitAcl::check($resource, 'update')): ?>
+    <?php include_component('repository', 'uploadLimit', array('resource' => $resource)) ?>
+  <?php endif; ?>
+
   <?php include_component('repository', 'holdings', array('resource' => $resource)) ?>
-<?php endif; ?>
+<?php endif; ?>
```

Comment:
Misty always used to be picky about this and I kind of got used to it! EOF blank line is missing! In Sublime you can enable `ensure_newline_at_eof_on_save` to handle that automagically.